04/app.js

```diff
@@ -1,8 +1,40 @@
 console.log('DOM');
 
 // struktura do wykorzystania w pętli
-const menuItems = [
-    {text: 'start', url: '/'},
-    {text: 'galeria', url: '/gallery'},
-    {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+const menuItems = [{
+        text: 'start',
+        url: '/'
+    },
+    {
+        text: 'galeria',
+        url: '/gallery'
+    },
+    {
+        text: 'kontakt',
+        url: '/contact'
+    },
+];
```

Comment:
Na podstawie rozwiązania z zadania 02, które Ci pokazałem spróbuj zrobić to zadania (jest dość podobne)